alicloud/table_alicloud_slb_load_balancer.go

```diff
@@ -0,0 +1,336 @@
+				Description: "The network type of the CLB instance. Valid values: internet|intranet.",
+			},
+			{
+				Name:        "v_switch_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The ID of the vSwitch to which the CLB instance belongs.",
+				Transform:   transform.FromField("VSwitchId"),
+			},
+			{
+				Name:        "vpc_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The ID of the virtual private cloud (VPC) to which the CLB instance belongs.",
+			},
+			{
+				Name:        "network_type",
+				Type:        proto.ColumnType_STRING,
+				Description: "The network type of the internal-facing CLB instance. Valid values: vpc|classic.",
+				Transform:   transform.FromField("ReleaseTime").Transform(transform.UnixToTimestamp),
+			},
+			{
+				Name:        "master_zone_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The ID of the primary zone to which the CLB instance belongs.",
+			},
+			{
+				Name:        "slave_zone_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The ID of the secondary zone to which the CLB instance belongs.",
+			},
+			{
+				Name:        "internet_charge_type",
+				Type:        proto.ColumnType_STRING,
+				Description: "The metering method of Internet data transfer. Valid values: paybybandwidth|paybytraffic.",
+			},
+			{
+				Name:        "create_time",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Description: "The time when the CLB instance was created. ",
+			},
+			{
+				Name:        "create_time_stamp",
+				Type:        proto.ColumnType_INT,
+				Description: "The timestamp when the instance was created.",
+			},
+			{
+				Name:        "pay_type",
+				Type:        proto.ColumnType_STRING,
+				Description: "The billing method of the CLB instance. Valid values: PayOnDemand.",
+			},
+			{
+				Name:        "resource_group_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The ID of the resource group.",
+			},
+			{
+				Name:        "address_ip_version",
+				Type:        proto.ColumnType_STRING,
+				Description: "The IP version. Valid values: ipv4 and ipv6.",
+				Transform: transform.FromField("AddressIPVersion"),
+			},
+			{
+				Name:        "modification_protection_status",
+				Type:        proto.ColumnType_STRING,
+				Description: "Indicates whether the configuration read-only mode is enabled for the CLB instance. ",
+			},
+			{
+				Name:        "modification_protection_reason",
+				Type:        proto.ColumnType_STRING,
+				Description: "The reason why the configuration read-only mode is enabled.",
+			},
+			{
+				Name:        "bandwidth",
+				Type:        proto.ColumnType_INT,
+				Description: "The maximum bandwidth of the listener. Unit: Mbit/s.",
+			},
+			{
+				Name:        "internet_charge_type_alias",
+				Type:        proto.ColumnType_STRING,
+				Description: "The metering method of Internet data transfer. Valid values: paybybandwidth|paybytraffic.",
```

Comment:
Is the description correct? This looks similar to the description of the `internet_charge_type` column.

Reply:
In doc this API does not return this data but SDK returns. Updated the description